src/v0/destinations/criteo_audience/config.js

```diff
@@ -0,0 +1,10 @@
+const version = "2022-10";
+const BASE_ENDPOINT = `https://api.criteo.com/${version}/`;
+const operation = ["add", "remove"];
+
+const MAX_IDENTIFIERS = 50000;
```

Comment:
Provide doc reference here.

Reply:
So the case is, in docs, it is not mentioned but when I type the same on Google we can see the limit of 50000 and we did testing around the same and found it to be the 
![Uploading Screenshot 2023-01-30 at 11.45.32 AM.png…]()
correct limit. Attaching the SS for Ref